starboard/android/shared/player_components_factory.cc

```diff
@@ -44,7 +44,11 @@ bool PlayerComponents::Factory::OutputModeSupported(
       return true;
     }
     DrmSystem* android_drm_system = static_cast<DrmSystem*>(drm_system);
-    return !android_drm_system->require_secured_decoder();
+    bool is_widevine_l1 = android_drm_system->require_secured_decoder();
```

Comment:
nit: name it "require_secure_decoder" to be in sync with the function name, also the message below should be "due to secure decoder required".

Reply:
Done.